source/includes/v1/records_api.md

```diff
@@ -219,6 +220,7 @@ Parameter            | Description
 `fields`             | Restricts field to retrieve. This parameter accepts multiple field names separated by commas. Example: `fields=field1,field2,field3`
 `pretty_print`       | If set to true (default is false), pretty prints JSON and JSONP output
 `format`             | Format of the response output. Can be `csv` (default), `json`, `jsonp`, `geojson`, `geojsonp`
+`csv_separator`      | If the `format` is `csv`, defines the delimiter used. Can be `;` (default), `,`, <code>&#124;</code>, or the tabulation character `%09`.
```

Comment:
it can be any char on the ASCII table, it just has to be URL-encoded if it's a special char like a non-printing char

Reply:
Yes, I figured we'd mention the ones that may be used so that the doc is clearer. We could keep the explicit list and add "or any ASCII character" and a small note on URL encoding maybe?

Reply:
Changed to
```
If the `format` is `csv`, defines the delimiter used. Can be `;` (default), `,`, <code>&#124;</code>, or any ASCII character. Special characters however may require URL encoding.
```